lib/services/user.service.js

```diff
@@ -0,0 +1,104 @@
+import { v4 as uuidv4 } from 'uuid';
+import userLoader from '../loaders/user';
+
+export default class UserService {
+    constructor() {
+        this.users = userLoader();
+    }
+
+    all() {
+        return this.users
+            .filter(user => user.isDeleted === false)
+            .map(user => {
+                const { ...copy } = user;
+                delete copy.password;
```

Comment:
You are using delete password and isDeleted flag multiple times, I would suggest to have separate function like `sanitize`

Reply:
Done